.drone.star

```diff
@@ -232,8 +232,11 @@ def gui_test_pipeline(ctx):
     return pipelines
 
 def build_client(image = OC_CI_CLIENT, ctest = True):
-    cmake_options = '-G"%s" -DCMAKE_C_COMPILER="%s" -DCMAKE_CXX_COMPILER="%s" -DCMAKE_BUILD_TYPE="%s"'
-    cmake_options = cmake_options % (build_config["generator"], build_config["c_compiler"], build_config["cxx_compiler"], build_config["build_type"])
+    cmake_options = '-G"%s"' % build_config["generator"]
+
+    if image != OC_CI_SQUISH:
```

Comment:
Why the change here?

Reply:
For [ctest pipeline](https://drone.owncloud.com/owncloud/client/18160/4/2), I am using `owncloudci/client:fedora-39-amd64` which doesn't have to craft environment. so maintaining the previous build configs if not using `OC_CI_SQUISH` which has craft environment